.github/workflows/delete_gitlab_branch.yaml

```diff
@@ -0,0 +1,21 @@
+name: Delete removed branch from GitLab mirror
+
+on:
+  delete:
+    branches-igore:
```

Comment:
Igor? This guy?

<img src="https://static.wikia.nocookie.net/ultimatepopculture/images/3/3a/Igor_Young_Frankenstein.jpeg/revision/latest?cb=20200807232901" width="400" />

Reply:
Good catch :) Fixed.